JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -17,6 +17,8 @@ module Microsoft.ApplicationInsights {
         accountId: () => string;
         sessionRenewalMs: () => number;
         sessionExpirationMs: () => number;
+        properties: () => Object;
+        measurements: () => Object;
```

Comment:
I still think (yeah I mentioned this already many times :)) that since you're adding onInitializeTelemetry, this other mechanism of adding properties and measurements is redundant - since you can do the same with onInitializeTelemetry